cirq-ft/cirq_ft/infra/testing.py

```diff
@@ -0,0 +1,133 @@
+# Copyright 2023 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from dataclasses import dataclass
+from typing import Any, Dict, List, Sequence, Tuple
+
+import cirq
+import numpy as np
+from cirq._compat import cached_property
+from cirq_ft.infra import gate_with_registers, t_complexity_protocol
+from cirq_ft.infra.decompose_protocol import _decompose_once_considering_known_decomposition
+
+
+@dataclass(frozen=True)
+class GateHelper:
```

Comment:
Likewise, should this be in a testing directory to stay consistent with cirq folder structure?

Reply:
ditto as commented above. Let's punt the directory reorganization to a later point once the library has grown bigger in size.